src/locales/en-US/commands/history.json

```diff
@@ -123,7 +123,11 @@
 	"informationTeryl": [
 		"Our oldest Archivist, from Sector 0A, guards Earth and its Solar System's colonies.",
 		"Teryl — and by extension Sector 0A — has a dedicated alert level, named **Purple Alert**, which is designed to call all the reinforcements needed to divert any enemy attacks within a single second. It was called only once, as Skyra charged to the battlefield while unleashing an attack so powerful it nearly irreversibly vaporized her heart.",
+		"Teryl has over 40000 language models from Earth. Such repository also includes historical and constructed languages.",
+		"Teryl likes to give visitors a warm welcome, offering tours around the Library and the Archive, as well as questions to pick the most suitable item specific to the user's tastes based on their answers and reactions. He often takes wild guesses when recommending, but they are also very spot on and spark the visitor's curiosity.",
 		"Teryl's body was destroyed long ago, what we see in his library is just a solid light hologram of him. He is not bothered by this, though, as it allows him to be in multiple places at the same time.",
+		"Teryl's library includes all the historical data from Earth that could be read since 2070 — the year when he started mirroring all data into his Archive — and also includes studies of his own based on related information.",
```

Comment:
```suggestion
		"Teryl's library includes all the historical data from Earth that could be read in the year 2070 — the year when he started mirroring all data into his Archive — and also includes studies of his own based on related information.",
```

Reply:
Is that right? Teryl's library includes all the reachable/readable data in the year 2070 (which includes past works) and also all the data he has collected over the millions of years he's been operative for

Reply:
that's the problem. Your version claimed that content made in 2071 would not be readable. By applying my change you do include <= 2070 and >= 2070

Reply:
And the sentence "the year when he started mirroring all data" already implies that it is an ongoing process.